src/frontend/Sider/Oppgavebenk/typer/oppgavetema.ts

```diff
@@ -43,6 +45,18 @@ export const oppgaveTypeTilTekst: Record<Oppgavetype, string> = {
     VUR_SVAR: 'Vurder svar',
 };
 
+export const oppgaveTypeTilVisningstekstSomTarHensynTilKlage = (
+    oppgavetype: Oppgavetype,
+    oppgaveBehandlingstype?: OppgaveBehandlingstype
+): string => {
+    if (oppgavetype === 'BEH_SAK') {
+        if (oppgaveBehandlingstype === OppgaveBehandlingstype.Klage) {
+            return 'Behandle sak (klage)';
+        }
+    }
```

Comment:
Synes dette er litt ryddigere 😄 
```suggestion
    if (oppgavetype === 'BEH_SAK' && oppgaveBehandlingstype === OppgaveBehandlingstype.Klage) {
          return 'Behandle sak (klage)';
    }
```